microservice/challenge-1/ReminderAppl/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM openjdk:8-jdk-alpine as basebuilder
+
+RUN mkdir -p /app/src
+COPY . /app/src
+WORKDIR /app/src
+RUN ./mvnw clean package
```

Comment:
this step fails to run locally, mvnw is not part of the binaries I have locally

Reply:
Added the maven wrapper specific files